resources/css/index.css

```diff
@@ -19,193 +18,167 @@ body, html {
   padding: 0;
 }
 
-
 .wrapper {
   margin: 0 auto;
   width: 1600px;
 }
 
-.background-overlay .container .main-title {
-  margin-bottom: 30px;
-}
-
-.background-1 {
-  background-image: url(../images/background/Bg.png);
-  background-position: 50% 0px;
-  width: 1600px;
-  height: 750px;
-  position: absolute;
-  background-attachment: fixed;
-  top: 0px;
-}
-
-.background-overlay .container {
-  width: 970px;
-  margin: 0 auto;
-  text-align: center;
-  position: relative;
-  margin-top: 130px;
-}
-
-.container .header {
-  display: block;
-  float: left;
-  text-transform: uppercase;
-  font-size: 36px;
-  font-weight: 700;
-  color: #fff;
-}
-
-.container .divider {
-  display: block;
-  margin: 0 auto;
-}
-
-.container .links a {
-  display: inline-block;
+.button {
   box-sizing: border-box;
   border: 1px solid black;
-  width: 230px;
-  padding-bottom: 15px;
-  text-transform: uppercase;
-  text-align: center;
-  text-decoration: none;
-  color: #fff;
   background-color: rgba(0, 0, 0, 0.6);
-  padding-top: 16px;
-  margin-right: 25px;
-  margin-top: 32px;
+  color: #fff;
+  display: inline-block;
   font-size: 18px;
   font-family: "Lato", sans-serif;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px 35px;
+  margin-top: 30px;
 }
 
-.container .links a:last-child {
-  margin: 0;
-  border: 1px solid #fff;
-  background-color: rgba(255, 255, 255, 0.2);
- }
-
-.container .links {
-  display: inline-block;
+.button-book {
+  margin-right: 30px;
 }
 
-.shadow {
-  box-shadow: 0px 50px 119px 0px rgba(0,0,0,0.67);
-  width: 1600px;
-  height: 160px;
-  top: -160px;
-  position: absolute;
-  z-index: 1;
-}
-
-.container .main-title {
-  font-family: "Yeseva One", sans-serif;
-  display: block;
-  text-align: center;
-  font-size: 90px;
-  color: #fff;
-  margin: 0;
-  font-weight: 400;
-  text-shadow:  1px 1px 2px #000;
+.button-menu {
+  border: 1px solid #fff;
+  background-color: rgba(255, 255, 255, 0.2);
+  margin-left: -5px;
 }
 
 .container {
-  position: relative;
-  z-index: 2;
+  background-attachment: fixed;
```

Comment:
I don't think it's necessary, imho

Reply:
I think it adds some dynamic effects at the page.